### PR TITLE
Simulation Orchestration Concern 

### PR DESCRIPTION
**Pull request by scopatz**
This is a modification to CEP2 that allows for something like Rickshaw

### CODE REVIEW

**Comment by scopatz** [comment]
Note that this was discussed at the Cyclus community meeting in July.

File: source/cep/cep2.rst [hunk 1]
```diff
@@ -137,58 +143,78 @@ which are pertinent to fuel cycle calculations:
...
 resource exchange put it with the domain models to be safe.
 
-The |cyclus| development team currently provides and supports three projects, 
-one for each concern:
+The |cyclus| development team currently supports projects for these concerns:
 
 * `Cyclus`_ - Dynamic Resource Exchange Simulation (individual actors, discrete
   time, canonical supply chain entities)
 * `Cycamore`_ - Domain Models
-* `Cyclist`_ -  Analysis & Visualization
```
Comment (gonuke) [inline source/cep/cep2.rst]: Should we somehow recognize `Cyclist` as an historical artifact?

**Comment by scopatz** [comment]
> Related: do we have a CEP status for "Approved with pending modification", in part so that we can publish the site with the modified CEP? (although raises questions about publishing the diff, etc... hmmm)

Nope, there is nothing like that in CEP1 - but we could add it, if desired.

File: source/cep/cep2.rst (hunk 1 above)
Comment (scopatz) [reply source/cep/cep2.rst]: Sure, I can add it back and put `(historical)` after it

Comment (scopatz) [reply source/cep/cep2.rst]: Done!

File: source/cep/cep2.rst [hunk 2]
```diff
@@ -276,7 +308,9 @@ References and Footnotes
 
 .. _Cyclus: https://github.com/cyclus/cyclus
 .. _Cycamore: https://github.com/cyclus/cycamore
-.. _Cyclist: https://github.com/cyclus/cyclist2
+.. _Cymetric: https://github.com/cyclus/cymetric
+.. _Cyclist: https://github.com/cyclus/cyclist
```
Comment (gonuke) [inline source/cep/cep2.rst]: URL is `cyclist2`

Comment (scopatz) [reply source/cep/cep2.rst]: OOps Thanks! Fixed

**Comment by bam241** [comment]
thx @scopatz !

everybody seems to agree... so I am merging

**Comment by scopatz** [comment]
Thanks All!

**Pull request merged**